src/plugin.js

```diff
@@ -68,22 +58,41 @@
     });
   });
 
-  oldOptions = videojs.Player.prototype.options;
-  videojs.Player.prototype.options = function() {
-    var options = oldOptions.call(this);
-    if (Object.prototype.toString.call(options.children) === '[object Array]') {
-      for (var i = 0; i < options.children.length; i++) {
-        var childName = options.children[i];
+  // The `children` property of `options_` used to be an object, but it is
+  // now an array. This restores the old behavior of that property (while
+  // retaining array behavior).
+  Component.prototype.options5_ = Component.prototype.options;
```

Comment:
Oh, you meant to call it `options5`? Can we have a more obvious name that doesn't look like a typo?